src/samples/ThumbnailsOpenChannelSample.js

```diff
@@ -0,0 +1,585 @@
+import { useState, useRef } from 'react';
```

Comment:
Would be nice to keep the file and component convention of OpenChannel<SampleName>